packages/vaults/src/transaction/config/purge.ts

```diff
@@ -8,7 +8,12 @@ import {TransactionMapperAbstract} from "../transaction_mapper";
 import {TransactionRequest} from "../transaction_request";
 import {RequestMapperAbstract} from "../request_mapper";
 
-
+/**
+ * This type of transaction is used to clear the queue.
```

Comment:
This is a bit of a complicated method. Maybe some specific instruction on when to use it will be helpful, like:

Use this method to clear all transactions from the current queue of those awaiting approval. It is meant to be used only in emergency situations if something unexpected happens and transactions end up stuck in a non-approvable state. Can be requested/approved only by admins.